test/clj/cider/nrepl/middleware/util/instrument_test.clj

```diff
@@ -170,3 +170,7 @@
          '#{[(set! (. inst field) (bp (bar) [2])) []] [(bar) [2]]}))
   (is (= (breakpoint-tester '(set! (.field inst) (bar)))
          '#{[(set! (. inst field) (bp (bar) [2])) []] [(bar) [2]]})))
+
+(deftest instrument-deftest
+  (is (= (breakpoint-tester '(deftest foo (bar)))
+         '#{[(bar) [2]]})))
```

Comment:
@Malabarba, do you by any chance have a guess why this test behaves differently depending on how it is run? If I run from cider (C-c C-,), it passes as-is. If I run "lein with-profile +test-clj test", it fails, expecting a different result (https://travis-ci.org/clojure-emacs/cider-nrepl/jobs/76097543#L224). I have no clue why.